src/trunk.c

```diff
@@ -2300,9 +2314,10 @@ trunk_bundle_live_for_pivot(trunk_handle *spl,
                             uint16        pivot_no)
 {
    debug_assert(pivot_no < trunk_num_children(spl, node));
-   trunk_bundle *bundle    = trunk_get_bundle(spl, node, bundle_no);
-   uint16        branch_no = trunk_bundle_start_branch(spl, node, bundle);
-   return trunk_branch_live_for_pivot(spl, node, branch_no, pivot_no);
+   return trunk_bundle_in_range(spl,
+                                bundle_no,
+                                trunk_pivot_start_bundle(spl, node, pivot_no),
+                                trunk_end_bundle(spl, node));
```

Comment:
I don't have enough data-structural context on these bundle handles, but finding the use of `trunk_end_bundle` alongside `trunk_pivot_start_bundle` raised a doubt as to whether we are mixing-up interfaces here?

Would it be too wrong, or too misleading, to add a wrapper `trunk_pivot_end_bundle`, aliased to `trunk_end_bundle`, adding a prolog-comment to the new fn explaining why pivot_end is the same as end_bundle?

This way, at least, to the casual reader of this code, the calls will appear 'symmetric'.

Reply:
I understand your point and concern here, but wouldn't that interface imply that pivots have their own end bundles?

This is another place where the medium-term fix will be that the node format will change.